tests/integration/030_approval_policy/main.tf

```diff
@@ -0,0 +1,27 @@
+provider "random" {}
```

Comment:
general question - is updating a resource ever tested in the provider?

Reply:
yes. In the acceptance tests.